### DISCUSSION TITLE
Have a go at using MXToolbox's Header Analyser - Much clearer results

### DISCUSSION THREAD

**Discussion by Gregory-EliotSinclair** [microsoft/MHA#667] [Ideas]
Not sure where to put feedback, I followed a link on the MHA page.

The result from MHA is a wall of text that takes more effort than it needs to to get a result. MXT has the best HA tool out there and it wouldn't be difficult to replicate. Would make this tool far more usable.

**Comment by Gregory-EliotSinclair** [comment]
Closed with no comment. Shot M$, keep up the good work. My expectations remain at basement level.

**Reply by stephenegriffin** [reply]
What comment were you expecting? You basically asked me to go copy someone else's UI. If you have specific feedback on MHA, please give it.
